metals/src/main/scala/scala/meta/internal/metals/ammonite/Ammonite.scala

```diff
@@ -246,13 +246,22 @@ final class Ammonite(
       case (command, script) =>
         val extraScripts = buffers.open.toVector
           .filter(path => path.isAmmoniteScript && path != script)
+        val jvmOpts = userConfig().ammoniteJvmProperties.getOrElse(
```

Comment:
The priority is the ammonite options specified in the configuration, otherwise we use JVM_OPTS variable and `.jvmopts` file.

This is the same as in the case of Metals server. I wonder whether we should use jvmopts, maybe just the settings?

Reply:
I would keep it as simple as possible and only use UseConfig.
You don't have to worry what if I have `.jvmopts` but don't want to use if for ammonite, is `.jvmopts` prioritized over JVM_OPTS etc.

Also I think JVM_OPTS are only used by CI and those systems, users normally specify JAVA_OPTS (e.g. `amm` script respects JAVA_OPTS not JVM_OPTS).

Reply:
I remove the alternatives and using only user config now.